packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -305,6 +313,32 @@ const main = async () => {
         .addOption(
             new program.Option('-w, --wait', 'wait for the deployment to complete before exiting')
         )
+        .addOption(
+            new program.Option(
+                '-u, --user <email>',
+                'the e-mail address you used to register with Managed Runtime',
+                (val) => {
```

Comment:
Parser as a third parameter is only works for `command.option()`. For `new program.Option()`, you gotta chain with `.argParser`.